packages/cursorless-vscode/src/ide/vscode/VscodeIDE.ts

```diff
@@ -52,6 +53,9 @@ export class VscodeIDE implements IDE {
     this.flashHandler = new VscodeFlashHandler(this, this.highlights);
     this.capabilities = new VscodeCapabilities();
     this.editorMap = new WeakMap<vscode.TextEditor, VscodeTextEditorImpl>();
+    if (this.runMode === "production") {
```

Comment:
if we're only creating channel in production, why don't we just use `console.log` below when it's null, and avoid needing checks in `NormalizedIde` or `Debug`

Reply:
We could do that

Reply:
I would argue for this